macros/src/command/mod.rs

```diff
@@ -57,6 +57,226 @@ pub struct CommandArgs {
+                group_args.required_permissions.clone()
+            } else {
+                self.required_permissions.clone()
+            },
+            required_bot_permissions: if Self::is_default(&self.required_bot_permissions) {
+                group_args.required_bot_permissions.clone()
+            } else {
+                self.required_bot_permissions.clone()
+            },
+            owners_only: if Self::is_default(&self.owners_only) {
+                group_args.owners_only
+            } else {
+                self.owners_only
+            },
+            guild_only: if Self::is_default(&self.guild_only) {
+                group_args.guild_only
+            } else {
+                self.guild_only
+            },
+            dm_only: if Self::is_default(&self.dm_only) {
+                group_args.dm_only
+            } else {
+                self.dm_only
+            },
+            nsfw_only: if Self::is_default(&self.nsfw_only) {
+                group_args.nsfw_only
+            } else {
+                self.nsfw_only
+            },
+            identifying_name: self.identifying_name.clone(), // `GroupArgs` doesn't have `identifying_name`
+            category: if Self::is_default(&self.category) {
+                group_args.category.clone()
+            } else {
+                self.category.clone()
+            },
+            custom_data: if Self::is_default(&self.custom_data) {
+                group_args.custom_data.clone()
+            } else {
+                self.custom_data.clone()
+            },
+            global_cooldown: if Self::is_default(&self.global_cooldown) {
+                group_args.global_cooldown
+            } else {
+                self.global_cooldown
+            },
+            user_cooldown: if Self::is_default(&self.user_cooldown) {
+                group_args.user_cooldown
+            } else {
+                self.user_cooldown
+            },
+            guild_cooldown: if Self::is_default(&self.guild_cooldown) {
+                group_args.guild_cooldown
+            } else {
+                self.guild_cooldown
+            },
+            channel_cooldown: if Self::is_default(&self.channel_cooldown) {
+                group_args.channel_cooldown
+            } else {
+                self.channel_cooldown
+            },
+            member_cooldown: if Self::is_default(&self.member_cooldown) {
+                group_args.member_cooldown
+            } else {
+                self.member_cooldown
+            },
+        }
+    }
+}
+
+/// Representation of the group attribute arguments (`#[group(...)]`)
+///
+/// Same as CommandArgs, but with some removed, because they wouldn't make sense
+#[derive(Default, Debug, darling::FromMeta)]
+#[darling(default)]
+pub struct GroupArgs {
+    prefix_command: bool,
+    slash_command: bool,
+    context_menu_command: Option<String>,
+
+    // When changing these, document it in parent file!
```

Comment:
Are these actually documented anywhere?

Reply:
I copied that from `CommandArgs`, I'm not sure what it really refers to.

Reply:
The second option would be preferred